pkg/datapath/iptables/iptables.go

```diff
@@ -297,6 +298,22 @@ func (m *IptablesManager) Init() {
 			"ip6tables kernel modules could not be loaded, so IPv6 cannot be used")
 		haveIp6tables = false
 	}
+
+	ipv6Disabled, err := os.ReadFile("/sys/module/ipv6/parameters/disable")
+	if err != nil {
+		if option.Config.EnableIPv6 {
+			log.WithError(err).Fatal(
+				"IPv6 is enabled but IPv6 kernel support could not be probed")
+		}
+		log.WithError(err).Warning(
+			"Unable to read /sys/module/ipv6/parameters/disable, disabling IPv6 iptables support")
+		haveIp6tables = false
+	} else if strings.TrimSuffix(string(ipv6Disabled), "\n") == "1" {
+		log.Debug(
```

Comment:
I think we should show this message to users. Maybe it should be a Warning?

Reply:
I'm following the same pattern as above (i.e. emitting a debug logline in case we fail to find/load ip6tables modules):
https://github.com/cilium/cilium/blob/78d075d1bedf7f6e3890dd2a9fab20383201f3c4/pkg/datapath/iptables/iptables.go#L290-L297